consensus/view_change.go

```diff
@@ -389,12 +348,12 @@ func (consensus *Consensus) onViewChange(msg *msg_pb.Message) {
 	consensus.viewIDBitmap.SetKey(recvMsg.SenderPubkey, true) // Set the bitmap indicating that this validator signed.
 	utils.Logger().Debug().
 		Int("numSigs", len(consensus.viewIDSigs)).
-		Int("needed", consensus.Quorum()).
+		Int64("needed", consensus.QuorumThreshold().Int64()).
 		Msg("[onViewChange]")
 
 	// received enough view change messages, change state to normal consensus
-	if len(consensus.viewIDSigs) >= consensus.Quorum() {
-		consensus.mode.SetMode(Normal)
+	if int64(len(consensus.viewIDSigs)) >= consensus.QuorumThreshold().Int64() {
```

Comment:
Why change int to int64? It seems not necessary

Reply:
Because staking code uses Decimal in `core/numeric` and more uniform to pick larger int that can accommodate old & new code

Reply:
based on the limitation of PBFT, the number of signatures will not too large. for core/numeric, the value is most used for float number, here the quorum is independent of the staking related value

Reply:
Understood; point was to have a uniform interface because Threshold in staking could be a big value, hence it uses big.Int, so not so much that its about number of signers, but more about threshold of staking amount.

Reply:
If using a uniform interface, then that's fine